apps/helsepersonell-api/src/main/java/no/nav/registre/testnorge/helsepersonell/produser/HelsepersonellController.java

```diff
@@ -0,0 +1,47 @@
+package no.nav.registre.testnorge.helsepersonell.produser;
```

Comment:
Vi har vel i de andre appene kalt modulen som inneholder controllerne for "provider". Kanskje endre til det?